.github/workflows/playwright_tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Playwright Tests
+
+on: 
+  push:  
+    branches: '**'
+
+  pull_request:
+    branches: [ "V2", "V3" ]    
+
+concurrency:
+  # Support push/pr as event types with different behaviors each:
+  # 1. push: queue up builds
+  # 2. pr: only allow one run per PR
+  group: ${{ github.workflow }}-${{ github.event.type }}${{ github.event.pull_request.number }}
+  # If there is already a workflow running for the same pull request, cancel it
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+env:
+  NEXT_PUBLIC_PROJECT_ID: ${{ secrets.NEXT_PUBLIC_PROJECT_ID }}
+
+jobs:
+  e2e_tests:
+      runs-on: ubuntu-latest
+      timeout-minutes: 30
+      permissions:
+        contents: read
+        deployments: write
```

Comment:
curious: why do you need this?

Reply:
Great catch, 
that was leftover from researching with cloudflare
It was needed so the GitHub cloudflare action I was using so use GITHUB_TOKEN with the cloudflare api
Will be removed 🤘